src/lib/validationSchemas/sendFeedbackSchema.js

```diff
@@ -0,0 +1,52 @@
+import { z } from 'zod';
+import { EMAIL_REGEX, PHONE_REGEX } from '../consts';
+
+const SendFeedback = z
+  .object({
+    name: z
+      .string({
+        required_error: 'Name is required',
+        invalid_type_error: 'Name must be a string',
+      })
+      .trim()
+      .min(1, {
+        message: 'Name can not be empty',
+      })
+      .max(128, {
+        message: 'Name must not be longer than 128 characters',
+      }),
+    phone: z
+      .string({
+        required_error: 'Phone number is required',
+        invalid_type_error: 'Phone number must be a string',
+      })
+      .trim()
+      .refine(val => PHONE_REGEX.test(val), {
+        message: 'Please, enter phone number in format +380XXXXXXXXX',
+      }),
+    callMe: z.boolean({
+      required_error: 'callMe is required',
+      invalid_type_error: 'callMe must be a boolean',
+    }),
+    email: z
+      .string()
+      .trim()
+      .refine(val => EMAIL_REGEX.test(val) && val.length <= 320, {
+        message: 'Please, enter valid email',
+      }),
+    message: z
```

Comment:
```suggestion
    email: z
      .string()
      .trim()
      .email(),
```
We don't have to validate emails by a regular expression, we already have email validation buit-in in zod